app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs

```diff
@@ -43,6 +43,7 @@
     layer=(hash
       type=model.type
       name=model.name
+      scale=model.scale
```

Comment:
scale will not save to the model. look at [_mutateModel](https://github.com/Flexberry/ember-flexberry-gis/blob/feature-search-form-scale-filter/addon/controllers/new-platform-flexberry-g-i-s-layer-metadata-e.js#L64) method